src/promise-katas-2.js

```diff
@@ -25,17 +25,48 @@ fetch("joke", "question").then(<insert your callback function>)
 // 1 Create a function that uses the fetch function to make a request to the "food" URL and returns
 // the data - expected return value "Cheese" of type String
 
-const food = () => {};
+// THIS does work but uses the .then
+/* const food = async () => {
+  return await fetch("food")
+    .then(
+      value => { return value.data; }
+    );
+}; */
+
+//THIS works but does not use the .then
+const food = async () => {
+   const { data } = await fetch("food");
+   return data;
+ };
+
 
 // 2 Create a function that uses the fetch function to make a request to the "cats" URL and returns
 // a list of cats in alphabetical order - expected return value ["Bandit", "Berry", "Puss in boots", "Smokey"] of type Array
 
-const cat = () => {};
+const cat = async () => {
+  const promise = await fetch("cats");
+  console.log(promise.data);
+
+   return promise.data.cats.sort();
+  };
+  /* const cat = async () => {
+    const { data } = await fetch("cats");
+    console.log(data);
+
+   return data.cats.sort();
+  }; */
```

Comment:
When using the fetch method its always a good idea to use the `.then` method to chain the results into:
```
const cat = async () => {
  return fetch("cats").then((res) => {
    return res.data.cats.sort();
  });
};
```
Also, once you are done with using debugging type console logs like this its best to remove them as all they do is clutter up the test readout and the code generally